docs/advanced-usage.rst

```diff
@@ -448,3 +448,39 @@ To enable this simply define environment variable `SSLKEYLOGFILE`:
 
 Then configure the key logfile in `Wireshark <https://wireshark.org>`_, see
 `Wireshark TLS Decryption <https://wiki.wireshark.org/TLS#TLS_Decryption>`_ for instructions.
+
+Custom SSL Contexts
+-------------------
+
+.. versionadded:: 1.17.0
+
+It is possible to provide a specific `SSL Context objects`_ directly to urllib3
+to exercise fine-grained control over the SSL configuration used in urllib3.
+
+For the purposes of compatibility, if you intend to customise an ``SSLContext``
+object we *strongly* recommend you obtain one from the following factory
```

Comment:
We can remove "strongly". When this was written the alternative was to create your own SSLContext, which is insecure, but now the alternative is to use `ssl.create_default_context` which is OK.